packages/scalecube-services/src/services/helpers/serviceValidations.js

```diff
@@ -0,0 +1,46 @@
+export const isValidService = ( service ) => {
+  let isValid = true;
+  const meta = service.constructor.meta || service.meta;
+
+  if ( meta ) {
+
+    isValid = isContainServiceName(meta) && isContainMethods(meta);
+
+  } else {
+    isValid = false;
+  }
+
+  return isValid;
+};
+
+const isContainServiceName = ( meta ) => {
+  if ( typeof meta.serviceName !== 'string' ) {
+    console.error(new Error('Service missing serviceName:string'));
+    return false;
+  }
+  return true;
+};
+
+const isContainMethods = ( meta ) => {
+
+  if ( !meta.methods ) {
+    console.error(new Error('Service missing methods:object'));
+    return false;
+  }
+  if ( meta.methods && typeof meta.methods !== 'object' ) {
```

Comment:
`meta.methods` is redundant because on previous step there is a check for  `!meta.methods` and `return` statement
this check `typeof meta.methods !== 'object'` will return same result for Array as for an Object